Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -49,6 +49,9 @@
     price: 800
   - type: ResearchClient
   - type: TechnologyDatabase
+  - type: MaterialStorageMagnetPickup # Frontier
+    range: 0.30
+    magnetEnabled: false
```

Comment:
set these to false on default spawn and then we can add it

Reply:
The comp is
magnetEnabled: false

Reply:
Fixed it by making the default to false.